docs/addons/gmaestro.md

```diff
@@ -0,0 +1,67 @@
+# gMaestro add-on for Amazon EKS Blueprints
+
+The gMaestro Blueprints AddOn deploys the gMaestro Agent on Amazon EKS using the [eks-blueprints](https://github.com/aws-quickstart/cdk-eks-blueprints) [CDK](https://aws.amazon.com/cdk/) construct.
```

Comment:
I would rather remove this -  using the [eks-blueprints](https://github.com/aws-quickstart/cdk-eks-blueprints) [CDK](https://aws.amazon.com/cdk/) construct.
 and rephrase the sentence better to say what the addon actually does apart from deploying the agent on Amazon EKS. You need one or two sentence to explain what the addon does?